pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -580,10 +580,20 @@ func (m *kubeGenericRuntimeManager) computePodResizeAction(pod *v1.Pod, containe
 			currentMemoryLimit = kubeContainerStatus.Resources.MemoryLimit.Value()
 		}
 		if kubeContainerStatus.Resources.CPULimit != nil {
-			currentCPULimit = kubeContainerStatus.Resources.CPULimit.MilliValue()
+			kubeCurrentCPULimit := kubeContainerStatus.Resources.CPULimit.MilliValue()
```

Comment:
this code is a bit confusing, and could benefit from a comment & better variable names.

```suggestion
			runtimeReportedCPULimit := kubeContainerStatus.Resources.CPULimit.MilliValue()
```